ignite/contrib/handlers/neptune_logger.py

```diff
@@ -187,7 +187,10 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
 
         if kwargs.get("offline_mode", False):
             self.mode = "offline"
-            neptune.init(project_qualified_name="dry-run/project", backend=neptune.OfflineBackend())
+            neptune.init(
```

Comment:
@Blaizzy can you check if this is a correct fix ?

Reply:
I will take a look at it tomorrow morning :)